styles/index.css

```
@@ -0,0 +1,178 @@
+}
+.image-1 {
+    background-image: url(assets/bule.jpeg);
+}
+h1 {
+    font-family:'Courier New' ; 
+}
+h2 {
+    font-family:'Courier New' ; 
+    font-size: 2rem;
+}
+
+button {
+    background-color: rgb(236, 236, 244);
+    border-radius: 10px;
+    /* color: rgb(7, 7, 7); */
+    /* font-size:small; */
+    margin-top: 2rem;
+    padding: 10px;
+} */
+.lower {
+    color: #c86733;
+    margin-top: 7rem;
+    font-size: 100px;
+}
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+body {
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-size: cover;
+    color: #F0E6F0;
+    text-align: center;
+}
+header {
+    min-height: 100px;
+    color: rgb(16, 87, 132);
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-size: 30px;
+}
+.boxes {
+    display: grid;
+    justify-content: space-around;
+    align-items: center;
+    margin-left: 2rem;
+    margin-right: 2rem;
+}
+.top-box {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    padding: 1em;
+}
+.grid-container {
+    display: grid;
+    grid-template-columns: 300px 300px 300px;
+    grid-template-rows: 3fr 1fr;
+    color: rgb(255, 255, 255);
+    grid-gap: 50px;
+    justify-items: center;
+    align-items: center;
+    font-size: 2em;
+    padding-top: 40px;
+    padding-bottom: 20px;
+}
+
+.grid-item {
+    background-color:rgb(16, 87, 132);
+    border: 1px solid rgba(62, 75, 78, 0.383);
+    border-radius: 30px;
+    text-align: center;
+    /* padding: 10px; */
+    font-size: 25px;
+    width: 250px;
+    height: 250px;
+}
+.grid-item-1 {
+    background-color: rgba(98, 129, 117, 0.8);
+    border: 1px solid rgba(62, 75, 78, 0.383);
+    border-radius: 30px;
+    text-align: center;
+    padding: 10px;
+    font-size: 25px;
+    width: 1000px;
+    height: 250px;
+}
+footer {
+    color: rgb(255, 255, 255);
+    background-color: rgb(16, 87, 132);
+    font-size: large;
+    text-align: center;
+    position:relative;
+    bottom: 0;
+}
+
+button {
+	width: 40px;
+	height: 40px;
+	font-size: 1em;
+	line-height: 30px;
+    text-align:center;
+	border-radius: 10px;
+	outline: none;
+	border: none;
+	color: #fff;
+	cursor: pointer;
+	background: transparent;
+	box-shadow: inset -2px -2px 2px rgba(255,255,255,.2),
+				inset 2px 5px 15px rgba(0,0,0,.5);
+}
+
+button:active{
+	box-shadow: inset 2px 2px 2px rgba(255, 255, 255, 0.1),
+				inset -2px -5px 15px rgba(0,0,0,.5);
+}
+
+input{
+	border-radius: 10px;
+	outline: none;
+	border: none;
+	padding: 10px 0;
+	width: fit-content;
+	text-align: center;
+	font-size: 20px;
+	color: #fff;
+	margin: 0 10px;
+	background: transparent;
+	box-shadow: inset -2px -2px 2px rgba(255,255,255,.2),
+				inset 2px 5px 15px rgba(0,0,0,.5);
+}
+
+.yellow {
+    color:yellow
+}
+.red {
+    color:red;
+}
+.orange {
+    color:orange;
+}
+.green {
+    color:green;
+}
+.teal {
+    color:teal;
+}
```

More semantically meaningful class names would give you more flexibility.  Right now if you change the color scheme your class names no longer make sense:

```suggestion
.moderate {
    color:yellow
}
.hot {
    color:red;
}
.warm {
    color:orange;
}
.cool {
    color:green;
}
.cold {
    color:teal;
}
```